app/Http/routes.php

```diff
@@ -12,6 +12,7 @@
     Route::get('conferences/{conference}', 'ConferencesController@show');
 });
 
+Route::post('api/conferences/{conference}/introduction', 'ConferenceIntroductionController@store');
```

Comment:
Only reason it's outside of the main api group is because I need it unauthenticated for guest access